lightly/data/collate.py

```diff
@@ -304,3 +304,12 @@ def __init__(self,
             rr_prob=rr_prob,
             normalize=imagenet_normalize,
         )
+
+
+class SimSiamCollateFunction(SimCLRCollateFunction):
```

Comment:
I wouldn't introduce an extra class, if the settings are the exact same as for SimCLR. For example, we didn't write a new class for the MoCo v2 augmentations because they are exactly the same as SimCLR.

Reply:
Agreed - would prefer to keep these augmentations with the simclr name, or pull them into a more generic/high level class of base augmentations?

Reply:
I'd say keep them with the SimCLR name

Reply:
+1 for this advice, almost did this exact same thing (extend SimCLR for BYOL